src/SchemaVersion.cpp

```diff
@@ -35,27 +35,33 @@ SchemaVersion::SchemaVersion(int version,
 bool SchemaVersion::apply(const QSqlDatabase &db)
 {
     QSqlQuery q(db);
-    q.prepare(m_applySql);
-    if (q.exec()) {
-        QSqlQuery upgradeVer(db);
-        upgradeVer.prepare("update qtdbmigration_meta set cur_ver = cur_ver + 1");
-        return upgradeVer.exec();
-    } else {
-        return false;
+    for (QString query : m_applySql)
+    {
+        q.prepare(query);
```

Comment:
Ah, I see.

This won't work if several statements are inside? Good, than do this refactoring. Try to move this into separate commit (to give it why you are changing this).

Also, can you wrap all SQL statements into BEGIN/COMMIT block? It won't work on MySQL, but on other databases they will work.

Reply:
Added to a separate commit. SQL statements wrapped in transaction block.